concepts/basics/introduction.md

```diff
@@ -0,0 +1,111 @@
+# Introduction
+
+To set off the C++ journey we are starting with variables, function calls, and comments.
+
+## Comments
+
+Comments come in two flavors: single- and multi-line.
+Everything that comes after `//` on the same line is ignored by the compiler.
+Multi-line comments are also known as C-style comments. 
+They are surrounded by `/*` and `*/`.
+Anything that comes between these will be ignored as well.
+
+## Variables
+
+C++ is a typed language.
+All types need to be known at compile time, and so you generally need to state them explicitly.
+A variable's type cannot change.
+An integer variable with the name `years` can be declared like this:
+
+```cpp
+int years;
+```
+
+It is good practice to initialize variables upon declaration.
+C++ offers different mechanisms to do so.
+The version with the curly braces is more in line with modern C++, but the equal-sign version is also very common.
+
+```cpp
+int tomatoes{80};
+int potatoes = 40;
+```
+
+```exercism/caution
+C++ does allow using uninitialized variables.
+Until the variable is deliberately set, it is undefined and might contain anything.
+To avoid used-before-set errors and undefined behaviour it is adviseable to **always initialize**.
+```
+
+## Arithmetic Operations
+
+Arithmetic operators like `*`, `+`, or `-` can be part of an expression like `3 * 2` or `tomatoes + potatoes`. 
+
+## Updating Variables
+
+You can reassign variables, as long as they keep their type:
+
+```cpp
+tomatoes = tomatoes - 5; // tomatoes is now 75
+potatoes = (32 * 2) + 11; // potatoes is now 75 as well
+```
+
+## Functions
+
+Functions have a name, a return type and (possibly empty) parameter list.
+An example of a function named `always_fortyseven` that would always return 47 would look like this:
+
+```cpp
+int always_fortyseven() {
+    return 47;
+}
+```
+
+Here is `vip_fee`, which has one parameter:
+```cpp
+int vip_fee(int standard_fee) {
+    /*
+    vip_fee calculates the vip fee based on the standard_fee.
+    */
+    int vip_multi{3};
+    return standard_fee * vip_multi;
+}
+```
+
+Or `total_fee`, a function with three parameters and a call to another function.
```

Comment:
I don't know if this many examples of functions are needed, my suggestion would be to perhaps be to reduce to 2 examples.